### DISCUSSION TITLE
Havoc, erro clone DT

### DISCUSSION THREAD

**Discussion by VitorGSC** [ROM-builders/temporary#8551] [Q&A] [answered]
https://cirrus-ci.com/task/5064992599834624?logs=build#L30

I'm having trouble cloning my DT, I don't understand anymore

**Comment by Apon77** [comment, marked as answer]
> https://cirrus-ci.com/task/5064992599834624?logs=build#L30
> 
> I'm having trouble cloning my DT, I don't understand anymore

follow #tutorial or #local_manifests from t.me/rom_builders
